src/Content/Text/BBCode.php

```diff
@@ -451,6 +451,11 @@ public static function convertAttachment(string $text, int $simplehtml = self::I
 				$return = sprintf('<div class="type-%s">', $data['type']);
 			}
 
+			if ($embed && $data['player_url'] != '' && $data['player_height'] != 0) {
```

Comment:
maybe it's possible to add a unittest here for it:
https://github.com/friendica/friendica/blob/2025.07-rc/tests/src/Content/Text/BBCodeTest.php ?

like at https://github.com/friendica/friendica/blob/2025.07-rc/tests/src/Content/Text/BBCodeTest.php#L146

Reply:
Test is added.

Reply:
looks good =)